book/src/getting-started/2-our-first-cxx-qt-module.md

```diff
@@ -31,49 +32,57 @@ This is a lot to take in, so let's go one step at a time.
 Starting with the module definition:
 ```rust,ignore
 {{#include ../../../examples/qml_minimal/rust/src/cxxqt_object.rs:book_bridge_macro}}
+    // ...
```

Comment:
interesting hack to show just the top part :')

Reply:
Hehe :smiling_imp: